### PR TITLE
Add CLI link to global navigation

### PR DESCRIPTION
**Pull request by colebemis**
image

### CODE REVIEW

File: theme/src/primer-nav.yml [hunk 1]
```diff
@@ -10,6 +10,8 @@
       url: https://primer.style/css/tools/prototyping
     - title: Presentations
       url: https://primer.style/presentations
+    - title: CLI
```
Comment (colebemis) [inline theme/src/primer-nav.yml:13]: Do we want the title of this link to be `CLI` or `Command line`?

Comment (BinaryMuse) [reply theme/src/primer-nav.yml:13]: 🤔 🤔 🤔 Good question. I think CLI is fine but maybe others have other thoughts :)

**Review by BinaryMuse — APPROVED** [review]

File: theme/src/primer-nav.yml (hunk 1 above)
Comment (colebemis) [reply theme/src/primer-nav.yml:13]: cc @ampinsk

Comment (colebemis) [inline theme/src/primer-nav.yml:13]:
```suggestion
    - title: Command line
```

**Pull request merged**